samples/Chat/src/app/ChatHeader.tsx

```diff
@@ -27,17 +27,17 @@ export const ChatHeader = (props: ChatHeaderProps): JSX.Element => {
   const theme = useTheme();
 
   const leaveString = 'Leave';
+  const peopleString = 'People';
 
   return (
     <Stack horizontal={true} verticalAlign={'center'} horizontalAlign="end" className={chatHeaderContainerStyle}>
-      {/** FIXME: Similar to #1235 */}
-      <div className={paneButtonContainerStyle}>
-        <IconButton
-          onRenderIcon={() => (props.isParticipantsDisplayed ? <People20Filled /> : <People20Regular />)}
-          className={mergeStyles({ color: theme.palette.neutralPrimaryAlt })}
-          onClick={() => props.setHideParticipants(props.isParticipantsDisplayed)}
-        />
-      </div>
+      <IconButton
+        onRenderIcon={() => (props.isParticipantsDisplayed ? <People20Filled /> : <People20Regular />)}
+        className={mergeStyles(paneButtonContainerStyle, { color: theme.palette.neutralPrimaryAlt })}
+        onClick={() => props.setHideParticipants(props.isParticipantsDisplayed)}
+        aria-live={'polite'}
+        aria-label={peopleString}
```

Comment:
there should be a prop ariaLabel for IconButton component, but works too :)

Reply:
Staying consistent with the ones below for now.